geom/vecgeom/src/TGeoVGConverter.cxx

```diff
@@ -65,3 +115,20 @@ void TGeoVGConverter::ConvertGeometry()
    }
    printf("# Converted %d shapes to VecGeom ones\n", nconverted);
 }
+
+////////////////////////////////////////////////////////////////////////////////
+/// Select shape(s) for conversion.
+/// Conversion is performed only on selected shapes (if set)
+
+void TGeoVGConverter::SelectShapeType(TGeoShape::EShapeType shapeType)
+{
+   fSelectedShapeTypes.insert(shapeType);
+}
+
+////////////////////////////////////////////////////////////////////////////////
+/// Exclude shape(s) from conversion.
```

Comment:
Maybe add in the comment: "Excluded types have precedence in case a type is also selected."